timeseries/tests/smoketests/test_features_and_covariates.py

```diff
@@ -79,8 +79,8 @@ def test_predictor_smoke_test(
         "TemporalFusionTransformer": DUMMY_MODEL_HPARAMS,
         "Theta": {},
         # Override default hyperparameters for faster training
-        "AutoARIMA": {"max_p": 2},
-        "AutoETS": {"model": "AAA"},
+        "AutoARIMA": {"max_p": 2, "use_fallback_model": False},
```

Comment:
Why do we set `use_fallback_model` to `False`?

Reply:
We want to make sure that the tests catch the case when some models fail.

Before we had `use_fallback_model=True`, so even if AutoETS failed, we used the fallback model and the test passed. This is the reason why we didn't notice that the model fails if `len(time_series) < 2 * season_length`.

Reply:
Nice! Thanks!